classes/01class/Exercises/c01-e02/starkmatt/c01-e02.txt

```diff
@@ -0,0 +1,11 @@
+Excercise: C01-E02
+Student: Matt G
+
+Description
+
+I created two Alarms,
+One for Alarm for $5 and the other for $10
+
+Using North Virginia Region 
```

Comment:
Any specific reason for using US regions? The US region is indeed cheaper, though the latency is quite high due to the distance.

If you are sticking to the free tier (only one t2 or t3 micro instances) it won't cost you anything to run the VM on ap-southeast-2 (Sydney)